src/Uno.UI.Tests/Windows_UI_Xaml/Controls/When_Subclassed_ResourceDictionary_With_Custom_Property.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<UserControl
+    x:Class="Uno.UI.Tests.Windows_UI_Xaml.Controls.When_Subclassed_ResourceDictionary_With_Custom_Property"
```

Comment:
good idea to add this one :) make sure to give it a unit test since literal ResourceDictionary declarations (inside FrameworkElement.Resources) have a somewhat different code path